src/App.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react'
+const { ipcRenderer } = require('electron')
+
+const getFilePathFromEvent = (event) => {
+  return event.dataTransfer.files[0].path;
+}
+
+const getSubFolders = (file) => {
```

Comment:
We can change the parameter name. if we rename it to _folder_, it can be more nice

Reply:
I've missed it, you are right. Thanks!